### DISCUSSION TITLE
Milvus standalone - can vector data be stored in s3 with index and log files?

### DISCUSSION THREAD

**Discussion by GangaK-Draup** [milvus-io/milvus#22850] [Q&A and General discussion]
We have installed standalone milvus on EC2 machine. We made the necessary config changes and are able to store only log files and index files on S3, vector data is still on EC2

 Is this expected or can all the files be on S3 itself?

**Comment by LoveEachDay** [comment]
@ganga-1908 What kind of vector data is left on EC2? Are these are files under `/var/lib/milvus` directory?

**Reply by GangaK-Draup** [reply]
Yes these files are under ```/var/lib/milvus``` directory
```/var/lib/milvus/rdb_data``` takes up maximum amount of space and it contains files of .sst extension

We have mentioned storageType as minio (minio having s3 directory path and access keys) under ```common``` section, but the issue still persists

Adding the config files here for your reference
[docker-compose.3.txt](https://github.com/milvus-io/milvus/files/11017741/docker-compose.3.txt)
[milvus.3.txt](https://github.com/milvus-io/milvus/files/11017743/milvus.3.txt)

**Reply by LoveEachDay** [reply]
Those files are used for internal message queue (rocksmq) in milvus standalone. When you insert data into a collection, the data flows from milvus standalone --> rocksmq-->datanode-->minio. 
And you can change the configs under `rocksmq` to reduce the message queue data storage footprint.
